apis/v1/controllers/addresses_controller.js

```diff
@@ -133,8 +133,8 @@ exports.makeAddressReadable = (addresses) => {
               // ...addresses._doc,
               ...addresses,
                 status: statusKey,
-                parish: addresses.parish.parishName,
-                user_id: addresses.user_id.email,
+                parish: addresses.parish?.parishName || null,
```

Comment:
{
    "status": 201,
    "message": "Success.",
    "data": {
        "address_1": "FirstLine of Address",
        "address_2": "SecondLine of Address",
        "user_id": "tevinmarlon@mailme.com",
        "city": "WakesWook",
       **"parish": null,**
        "status": "PENDING",
        "_id": "6400f926e53975484d0c4608",
        "__v": 0
    },
    "error": null
}

that above is the response. it should be undefined not null.

Reply:
also as Hari mentioned we should remove unnecessary info from responses and add correct and clear response messages.
Therefore we should remove the address from the create update and delete since the user doesnt need to look at the data during those operations